testing/it/http/src/test/java/com/adobe/cq/wcm/core/components/it/http/ListIT.java

```diff
@@ -0,0 +1,52 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
```

Comment:
```suggestion
 ~ Copyright 2024 Adobe
```

Reply:
deleted the file, ComponentsIT test covers the case. It compares the list item HTML content with a content from an .html file